src/csvcubed/utils/qb/cube.py

```diff
@@ -17,6 +17,7 @@
     QbObservationValue,
     QbColumnStructuralDefinition,
 )
+from csvcubed.utils.sparql_handler.sparqlmanager import CSVWShape
```

Comment:
Can you store this elsewhere? I'd like to make sure that the `csvcubed build` functionality isn't dependent on `csvcubed inspect` functionality and vice versa. If you place it somewhere which isn't in one or the other we can avoid having any dependencies there.